ws-nextjs-app/pages/[service]/send/[id]/types.ts

```diff
@@ -115,6 +115,10 @@ export type PageProps = {
   pageData: {
     title: string;
     description: string;
+    settings: {
+      replyEmailAddress: string;
+      retentionPeriodDays: string;
+    };
```

Comment:
under the assumption these will always come through data, even if empty "" values are provided.

Reply:
Should we conditionally render the areas where these values are used? e.g: https://github.com/bbc/simorgh/blob/cc0aa6b393b2c75a2189249a1c5d5dc807f38719/ws-nextjs-app/pages/%5Bservice%5D/send/%5Bid%5D/SuccessScreen/index.tsx#L89

Should we do:
```
{ retentionPeriod && <Paragraph>{retentionPolicy}</Paragraph> }
```